frontend/app/src/store/blockchain/accounts/migrate.ts

```diff
@@ -0,0 +1,99 @@
+import { type Ref } from 'vue';
+import {
+  type Notification,
+  NotificationCategory,
+  Severity
+} from '@rotki/common/lib/messages';
+import { type MigratedAddresses } from '@/types/websocket-messages';
+import { useSessionAuthStore } from '@/store/session/auth';
+import { startPromise } from '@/utils';
+import { useNotificationsStore } from '@/store/notifications';
+
+export const useAccountMigration = defineStore(
```

Comment:
can we rename it to `useAccountMigrationStore`?

I have a template I created in WebStorm to generate the template for a store.
It still requires some manual interaction in nested ones but I can share it if you want.

Reply:
sure I can try the template